src/current/_includes/releases/v24.1/v24.1.0.md

```diff
@@ -325,7 +325,7 @@ COMMITTED</code> more easily, without needing to teach the schema changer about
     <tr>
         <td>
             <p class="feature-summary">SELECT FOR SHARE updates</p>
-            <p class="feature-description">A shared lock that is acquired explicitly using <a href="https://www.cockroachlabs.com/docs/stable/select-for-update#for-share-usage">SELECT FOR SHARE</a> or implicitly by a read-committed transaction, can now be re-acquired with higher strength by either using a SELECT FOR UPDATE statement or by writing to the key</p>
+            <p class="feature-description">A shared lock that is acquired explicitly using <a href="https://www.cockroachlabs.com/docs/v24.1/select-for-update#for-share-usage">SELECT FOR SHARE</a> or implicitly by a read-committed transaction, can now be re-acquired with higher strength by either using a SELECT FOR UPDATE statement or by writing to the key</p>
```

Comment:
```suggestion
            <p class="feature-description">A shared lock that is acquired explicitly using <a href="https://www.cockroachlabs.com/docs/v24.1/select-for-update#for-share-usage"><code>SELECT FOR SHARE</code></a> or implicitly by a read-committed transaction, can now be re-acquired with higher strength by either using a <code>SELECT FOR UPDATE</code> statement or by writing to the key.</p>
```
I didn't check all of them, but these are long enough that they should end in a period.